core/src/test/java/org/hibernate/ogm/backendtck/associations/order/OrderByQualifierTest.java

```diff
@@ -0,0 +1,66 @@
+/*
+ * Hibernate OGM, Domain model persistence for NoSQL datastores
+ *
+ * License: GNU Lesser General Public License (LGPL), version 2.1 or later
+ * See the lgpl.txt file in the root directory or <http://www.gnu.org/licenses/lgpl-2.1.html>.
+ */
+package org.hibernate.ogm.backendtck.associations.order;
+
+import static org.fest.assertions.Assertions.assertThat;
+
+import javax.persistence.OrderBy;
+
+import org.hibernate.ogm.utils.OgmTestCase;
+
+import org.junit.Test;
+
+/**
+ * Tests the use of {@link OrderBy} qualifier, on load association list.
+ *
+ * @author Fabio Massimo Ercoli
+ */
+public class OrderByQualifierTest extends OgmTestCase {
+
+	public static final String CONTRIBUTOR1 = "unclebob";
+	public static final String CONTRIBUTOR2 = "ddalto";
+	public static final String CONTRIBUTOR3 = "telegraph4eternity";
+	public static final String CONTRIBUTOR4 = "fax4ever";
+	public static final String CONTRIBUTOR5 = "phone4induction";
+
+
+	public static final String PROJECT_NAME = "Hibernate OGM";
+
+	@Test
+	public void testOrderByQualifier() {
+		OpenSourceProject project = new OpenSourceProject();
+		project.setName( PROJECT_NAME );
+
+		Programmer unclebob = new Programmer( CONTRIBUTOR1 );
+		Programmer ddalto = new Programmer( CONTRIBUTOR2 );
+		Programmer telegraph4eternity = new Programmer( CONTRIBUTOR3 );
+		Programmer fax4ever = new Programmer( CONTRIBUTOR4 );
+		Programmer phone4induction = new Programmer( CONTRIBUTOR5 );
+
+		// that's the original order:
+		project.addContributor( unclebob );
+		project.addContributor( ddalto );
+		project.addContributor( telegraph4eternity );
+		project.addContributor( fax4ever );
+		project.addContributor( phone4induction );
```

Comment:
Instead of having the Id as constants, why don't you create all the entities as constants? This makes the unit test shorter and easier to read